api/src/main/java/gov/va/vro/api/resources/BipResource.java

```diff
@@ -1,10 +1,6 @@
 package gov.va.vro.api.resources;
 
-import gov.va.vro.api.responses.BipClaimContentionsResponse;
-import gov.va.vro.api.responses.BipClaimResponse;
-import gov.va.vro.api.responses.BipClaimStatusResponse;
-import gov.va.vro.api.responses.BipContentionCreationResponse;
-import gov.va.vro.api.responses.BipContentionUpdateResponse;
+import gov.va.vro.api.responses.*;
```

Comment:
Do not use "*" form please. That is one the style check. Ian posted a fix for IntelliJ not to do this automatically if that is what you are using.

Reply:
Hmm... Intellij did it. Removed *.